tools/internal_ci/linux/grpc_e2e_performance_gke.sh

```diff
@@ -139,8 +139,8 @@ for language in "${disabledLanguages[@]}"; do
 done
 
 # Add arguments for languages.
-declare -a configLangArgs8core=()
-declare -a configLangArgs32core=()
+declare -a configLangArgs8core=("--category=scalable_unrestricted")
```

Comment:
This is backwards, right? `scalable_unrestricted` should contain the test that fails on 8-core machines.